src/tests/analyse.test.js

```diff
@@ -0,0 +1,18 @@
+import analyze from '../modules/analyse';
+
+test('Gets average', () => {
+  expect(analyze([1, 8, 3, 4, 2, 6]).average).toEqual(4);
+});
+
+test('Gets minimum value', () => {
+  expect(analyze([1, 8, 3, 4, 2, 6]).min).toEqual(1);
+});
+
+test('Full object', () => {
```

Comment:
- [ ] [OPTIONAL] In the next tests you make try to test all of the methods separately too, even if you test them at once as you did here.